thriftpy2/__init__.py

```diff
@@ -5,7 +5,7 @@
 from .hook import install_import_hook, remove_import_hook
 from .parser import load, load_module, load_fp
 
-__version__ = '0.4.11'
+__version__ = '0.4.12'
```

Comment:
Please do not modify the version, I will upgrade the version when the package releasing.

Reply:
Fixed in 8bb4ed6